src/pages/validator/amplifier/verifier-onboarding.mdx

```diff
@@ -6,192 +6,186 @@ import { Callout } from "/src/components/callout"
-
-## Onboard as a verifier
-
-### Run `tofnd`
+1. Run `tofnd`. [`ampd`](https://github.com/axelarnetwork/axelar-amplifier/tree/main/ampd#worker-onboarding), the off-chain daemon responsible for voting and signing with the amplifier protocol, needs access to a running `tofnd` instance in order to onboard as a worker or run the daemon. See the [tofnd repository](https://github.com/axelarnetwork/tofnd) for more info.
+
+2. Download `ampd` through either the [binary](https://github.com/axelarnetwork/axelar-amplifier/releases/tag/ampd-v0.1.0) or [docker](https://hub.docker.com/r/axelarnet/axelar-ampd).
+
+3. Create a `config.toml` file and put it in your `~/.ampd/` directory. 
+
+By default, `ampd` loads the `config` file from `~/.ampd/config.toml` when running any command. This can be overridden by passing `--config [path]`.
+
+The file should have the following format:
+
+```toml
+tm_jsonrpc=[JSON-RPC URL of Axelar node]
+tm_grpc=[gRPC URL of Axelar node]
+event_buffer_cap=[max blockchain events to queue. Will error if set too low]
+
+[service_registry]
+cosmwasm_contract=[address of service registry]
+
+[broadcast]
+batch_gas_limit=[max gas for a transaction. Transactions can contain multiple votes and signatures]
+broadcast_interval=[how often to broadcast transactions]
+chain_id=[chain id of Axelar network to connect to]
+gas_adjustment=[gas adjustment to use when broadcasting]
+gas_price=[gas price with denom, i.e. "0.007uaxl"]
+queue_cap=[max messages to queue when broadcasting]
+tx_fetch_interval=[how often to query for transaction inclusion in a block]
+tx_fetch_max_retries=[how many times to query for transaction inclusion in a block before failing]
+
+[tofnd_config]
+key_uid=[uid of key used for signing transactions]
+party_uid=[metadata, should just be set to ampd]
+url=[url of tofnd]
+
+# multisig handler. This handler is used for all supported chains.
+[[handlers]]
+cosmwasm_contract=[address of multisig contract]
+type="MultisigSigner"
+
+# message verifier handler. One per supported chain
+[[handlers]]
+chain_name=[chain name. Not necessary in the Sui case]
+chain_rpc_url=[URL of JSON-RPC endpoint for external chain]
+cosmwasm_contract=[verifier contract address]
+type=[handler type. Could be EvmMsgVerifier | SuiMsgVerifier]
+
+# workerset verifier handler. One per supported chain
+[[handlers]]
+chain_name=[chain name. Not necessary in the Sui case]
+chain_rpc_url=[URL of JSON-RPC endpoint for external chain]
+cosmwasm_contract=[verifier contract address]
+type=[handler type. Could be EvmMsgWorkerSetVerifier | SuiWorkerSetVerifier]
+```
 
-- If you are running `tofnd` for the first time, a verifier key will be generated for you automatically.
+Here's an example config for connecting to a local axelard node and `tofnd` process and verifying transactions from the Avalanche and Sui testnets:
```

Comment:
good to have the sample config with devnet-amplifier contract addresses and gateways

Reply:
https://github.com/axelarnetwork/axelar-contract-deployments/blob/e6f549de873e0ad211a073218f172d03f6e92c3b/axelar-chains-config/info/devnet-amplifier.json#L4